README.md

```diff
@@ -13,11 +13,11 @@ source directory, and then run colcon build in the usual manner:
     git clone https://github.com/atolab/rmw_cyclonedds
     cd ..
     rosdep install --from src -i
-    colcon build --cmake-args -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCYCLONEDDS_FROM_SOURCE=ON
+    colcon build
     export RMW_IMPLEMENTATION=rmw_cyclonedds_cpp
 
-If you prefer to install CycloneDDS yourself, set `-DCYCLONEDDS_FROM_SOURCE=OFF`. CMake caches this
-value, so you can omit it between builds
+Optionally, you can `git clone https://github.com/eclipse-cyclonedds/cyclonedds` in your workspace
+as well. If you don't, `rosdep` will download it from the package index.
```

Comment:
> If you don't, `rosdep` will download it from the package index.

Since that doesn't work on platforms which don't have the Debian package of CycloneDDS I think the instructions should rather mention a "full" from source build.

Reply:
Is there a reason why cyclonedds can't be released as a binary package and installable via rosdep outside of Debian? I thought rosdep's whole raison d'etre was to resolve dependencies in a cross-platform way.

Reply:
I am not refereeing to "technically impossible". But atm Debian packages are the only package-level binaries produced by the ROS buildfarm.

Reply:
gotcha.